model-api/bpmn-model/src/test/java/org/operaton/bpm/model/bpmn/DefinitionsTest.java

```diff
@@ -226,12 +213,7 @@ public void shouldAddMessageAndMessageEventDefinition() {
     startEvent.getEventDefinitions().add(anotherMessageEventDefinition);
 
     // validate model
-    try {
-      Bpmn.validateModel(bpmnModelInstance);
-    }
-    catch (ModelValidationException e) {
-      Assert.fail();
-    }
+    Bpmn.validateModel(bpmnModelInstance);
```

Comment:
Same here as in line 136

Reply:
I have done this refactoring intentionally.